lib/core/constants/event/event_constants.dart

```diff
@@ -4,19 +4,171 @@ class EventConstants {
 
   /// Guest limit 100
   static String defaultEventGuestLimit = '100';
+
+  static final List<Map<String, String>> timezoneOptions = [
```

Comment:
Could enhance later by defining Timezone entity, with text and value, rather than use Map